### PR TITLE
add test files to modules/canbus/vehicle/transit/protocol folder

### PR DESCRIPTION
**Pull request by sjiang2018**

### CODE REVIEW

File: modules/canbus/vehicle/transit/protocol/BUILD [hunk 1]
```diff
@@ -187,6 +202,18 @@ cc_library(
...
 
+cc_test(
+    name = "llc_motionfeedback2_21_test",
+    size = "small",
+    srcs = ["llc_motionfeedback2_21_test.cc"],
+    deps = [
+        ":llc_motionfeedback2_21",
+        "@gtest//:main",
+        "//modules/canbus/vehicle/transit/protocol:canbus_transit_protocol",
+
```
Comment (lianglia-apollo) [inline modules/canbus/vehicle/transit/protocol/BUILD]: remove emplty line.

File: modules/canbus/vehicle/transit/protocol/BUILD [hunk 2]
```diff
@@ -90,6 +90,21 @@ cc_library(
...
+cc_test(
+    name = "llc_auxiliaryfeedback_120_test",
+    size = "small",
+    srcs = ["llc_auxiliaryfeedback_120_test.cc"],
+    deps = [
+        ":llc_auxiliaryfeedback_120",
+        "@gtest//:main",
+        "//modules/canbus/vehicle/transit/protocol:canbus_transit_protocol",
+
```
Comment (lianglia-apollo) [inline modules/canbus/vehicle/transit/protocol/BUILD]: remove empty line

File: modules/canbus/vehicle/transit/protocol/BUILD [hunk 3]
```diff
@@ -219,4 +246,20 @@ cc_library(
...
+    size = "small",
+    srcs = ["llc_vehiclestatus_25_test.cc"],
+    deps = [
+        ":llc_vehiclestatus_25",
+        "@gtest//:main",
+        "//modules/canbus/vehicle/transit/protocol:canbus_transit_protocol",
+        
+    ],
+)
+
```
Comment (lianglia-apollo) [inline modules/canbus/vehicle/transit/protocol/BUILD]: remove all empty lines.

File: modules/canbus/vehicle/transit/protocol/llc_auxiliaryfeedback_120_test.cc [hunk 4]
```diff
@@ -0,0 +1,75 @@
+/******************************************************************************
+ * Copyright 2017 The Apollo Authors. All Rights Reserved.
```
Comment (lianglia-apollo) [inline modules/canbus/vehicle/transit/protocol/llc_auxiliaryfeedback_120_test.cc]: 2017 -> 2019

File: modules/canbus/vehicle/transit/protocol/llc_auxiliaryfeedback_120_test.cc [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+class llc_auxiliaryfeedback_120Test : public ::testing ::Test {
+ public:
+ protected:
+  virtual void SetUp() {
+    // setup parameters
+  }
+  Llcauxiliaryfeedback120 Llcauxiliary_feedback120_;
+};
+
+const std::uint8_t bytes = 0xFF;
```
Comment (lianglia-apollo) [inline modules/canbus/vehicle/transit/protocol/llc_auxiliaryfeedback_120_test.cc]: for const variables, please follow google coding style.

File: modules/canbus/vehicle/transit/protocol/llc_auxiliaryfeedback_120_test.cc [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+std::int32_t length = 1;
+
+TEST_F(llc_auxiliaryfeedback_120Test, inverter) {
+  EXPECT_EQ(Llcauxiliary_feedback120_.llc_fbk_inverter(&bytes, length), true);
+}
+
+TEST_F(llc_auxiliaryfeedback_120Test, ch) {
+  EXPECT_EQ(Llcauxiliary_feedback120_.llc_fbk_pdu_ch8(&bytes, length), false);
```
Comment (lianglia-apollo) [inline modules/canbus/vehicle/transit/protocol/llc_auxiliaryfeedback_120_test.cc]: use EXPECT_FALSE

File: modules/canbus/vehicle/transit/protocol/llc_motionfeedback2_21_test.cc [hunk 7]
```diff
@@ -0,0 +1,49 @@
+/******************************************************************************
+ * Copyright 2017 The Apollo Authors. All Rights Reserved.
```
Comment (lianglia-apollo) [inline modules/canbus/vehicle/transit/protocol/llc_motionfeedback2_21_test.cc]: 2017 --> 2019

File: modules/canbus/vehicle/transit/protocol/llc_motionfeedback2_21_test.cc [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+TEST_F(llc_motionfeedback2_21Test, motion_fdk) {
+  const std::uint8_t bytes = 0x00;
+  std::int32_t length = 1;
+  EXPECT_EQ(Llcmotionfeedback2_21.llc_fbk_vehiclespeed(&bytes, length), 0);
+  EXPECT_EQ(Llcmotionfeedback2_21.llc_motionfeedback2_counter(&bytes, length),
+            0);
+  EXPECT_EQ(Llcmotionfeedback2_21.llc_motionfeedback2_checksum(&bytes, length),
+            0);
+  EXPECT_EQ(Llcmotionfeedback2_21.llc_fbk_steeringrate(&bytes, length), 0);
+  EXPECT_EQ(Llcmotionfeedback2_21.llc_fbk_steeringangle(&bytes, length), 12.8);
```
Comment (lianglia-apollo) [inline modules/canbus/vehicle/transit/protocol/llc_motionfeedback2_21_test.cc]: use EXPECT_DOUBLE_EQ

File: modules/canbus/vehicle/transit/protocol/llc_motionfeedback2_21_test.cc [hunk 9]
```diff
@@ -0,0 +1,49 @@
...
+
+ protected:
+  Llcmotionfeedback221 motionfdk2_21_;
+};
+
+TEST_F(llc_motionfeedback2_21Test, motion_fdk) {
+  const uint8_t kData[8] = {0x9A, 0xFC, 0x56, 0xF7, 0x12, 0x34, 0xFF, 0xFF};
+  int32_t length = 1;
+
+  EXPECT_DOUBLE_EQ(motionfdk2_21_.llc_fbk_vehiclespeed(kData, length), 133.3);
```
Comment (Capri2014) [inline modules/canbus/vehicle/transit/protocol/llc_motionfeedback2_21_test.cc]: @sjiang2018 Different signals shall have different length, for instance , the llc_fbk_vehiclespeed has a length of 16 according to https://github.com/ApolloAuto/apollo/blob/0c9966e41561b8fcd4413da4054f9f048a27665b/modules/canbus/vehicle/transit/protocol/llc_motionfeedback2_21.cc#L55

File: modules/canbus/vehicle/transit/protocol/llc_vehiclestatus_25_test.cc [hunk 10]
```diff
@@ -0,0 +1,45 @@
...
+  virtual void SetUp() {}
+
+ protected:
+  Llcvehiclestatus25 Llcauxiliary_status25_;
+};
+
+TEST_F(llc_vehiclestatus_25Test, 12voltage) {
+  const std::uint8_t kBytes = 0xFF;
+  std::int32_t length = 1;
+  EXPECT_DOUBLE_EQ(Llcauxiliary_status25_.llc_fbk_12voltage(&kBytes, length),
```
Comment (Capri2014) [inline modules/canbus/vehicle/transit/protocol/llc_vehiclestatus_25_test.cc]: same issue

**Pull request merged**